Added default jmeter.properties to the plugin

foragerr opened the pull request:
1. Closes #38
   Added default jmeter.properties from jmeter 2.13 into plugin. 
   - If user sets `jmeterPropertyFile`, this value is used as jmeter properties
   - If user does not set , the jmeter.properties in `$projectDir/build/jmeter` is used
2. Closes #45 
   Set property `jmeter.save.saveservice.output_format=xml` in jmeterRun task, so results files are in xml format and can be transformed to HTML files for reports.

foragerr committed:
added resultsformat=xml in saveservice.properties

foragerr committed:
included default jmeter.properties

foragerr committed:
moved results=xml property from saveservice.properties into code

djangofan commented:
I assume that a jmeter.properties file that you put in  $srcDir/jmeter.properties  will get copied into $projectDir/build/jmeter at runtime?  If so, this sounds correct.

foragerr commented:
@djangofan no, if plugin user does not  set jmeterPropertyFile, then I'm defaulting to using the jmeter.properties that ships with jmeter version 2.13. 

Should I expect that there is always a jmeter.properties in srcDir?

foragerr committed:
changed order of jmeter.properties precedence

foragerr commented:
Ok, made another change based on what you said. Which jmeter.properties to use is now determined in this order:
1. Is jmeterPropertyFile defined in build.gradle? Then use it
2. Otherwise, look in $srcDir/test/jmeter and use jmeter.properties from there, if it exists
3. If neither, use default jmeter.properties - same as the one shipped with jmeter 2.13

foragerr commented:
@kulya can you please accept this pull-req or suggest any changes you want made?

vlaadbrain commented on src/main/groovy/com/github/kulya/gradle/plugins/jmeter/JmeterRunTask.java:
```diff
@@ -229,6 +248,7 @@ private String executeJmeterTest(String fileLocation) {
             specs.getSystemProperties().put("saveservice_properties", System.getProperty("saveservice_properties"));
             specs.getSystemProperties().put("upgrade_properties", System.getProperty("upgrade_properties"));
             specs.getSystemProperties().put("log_file", System.getProperty("log_file"));
+            specs.getSystemProperties().put("jmeter.save.saveservice.output_format","xml");
```
is this necessary, as i recall this used to be in the jmeter.properties file.

foragerr replied:
This is the line in jmeter.properties (it is commented out)

```
#jmeter.save.saveservice.output_format=csv
```

which implies that the default format is xml. But for some reason, when using this plugin to run a jmeter test, the results are in csv format. see #45. I added this line to remedy that.

Commit:
Adds suport for http://jmeter-plugins.org/ and extended report
generation. For now it generates all the reports.

- Bumped version number.

Commit:
Some code cleanup, made plugin addition backwards compatible. I.e. if an
existing project declares the plugins we will remove the in built ones.

Commit:
Addresses code review feedback:

- Make the extension optional, so the optional jars are only added when
  needed.
- Undid some layout changes.
- Updated documentation.

ecopoesis committed:
remove extraneous println

foragerr committed:
fixes https://github.com/kulya/jmeter-gradle-plugin/issues/27

foragerr committed:
added /bin and eclipse files to .gitignore

foragerr committed:
Merge remote-tracking branch 'ecopoesis/master'

Conflicts:
	src/main/groovy/com/github/kulya/gradle/plugins/jmeter/JmeterAbstractTask.groovy

foragerr committed:
Merge remote-tracking branch 'pokowaka/master'

Conflicts:
	build.gradle
	src/main/groovy/com/github/kulya/gradle/plugins/jmeter/JmeterPluginConvention.java
	src/main/groovy/com/github/kulya/gradle/plugins/jmeter/JmeterRunTask.java

foragerr committed:
Merge branch 'fix_#27'

Conflicts:
	build.gradle
	src/main/groovy/com/github/kulya/gradle/plugins/jmeter/JmeterAbstractTask.groovy

foragerr committed:
Minor Fixes

foragerr committed:
Fixed a problem with useSaveServiceFormat

foragerr committed:
Add pointer to new version of plugin.
